code/_onclick/item_attack.dm

```diff
@@ -95,7 +95,7 @@ avoid code duplication. This includes items that may sometimes act as a standard
 		var/mob/living/carbon/human/X = user
 		if(X.gloves && istype(X.gloves,/obj/item/clothing/gloves/force))
 			var/obj/item/clothing/gloves/force/G = X.gloves
-			power *= G.amplification
+			power *= 1
```

Comment:
Probably the worst way to deal with this. Remove the entire amplification code, and the var from the gloves as well.

Reply:
`amplification`, specifically? I only wanted to hit the playstyle of using e-sword with force gloves, not nerfing it entirely.